### PR TITLE
Chad's Changes

### PR DESCRIPTION
**Pull request by chad-daniels**
Chad's changes for integration with updated merge conflicts

### CODE REVIEW

**Comment by wanda-carlson** [comment]
Hi Chad, please notice the merge conflicts.

File: src/hello.py [hunk 1]
```diff
@@ -1,2 +1 @@
-# new file
-print("Hello")
```
Comment (chad-daniels) [inline src/hello.py:2]: Is this ok?

**Review by henry-richard — CHANGES_REQUESTED** [review]
I just dont like this

**Comment by chad-daniels** [comment]
Thanks for merging @wanda-carlson and @henry-richard , appreciate it!

**Pull request merged**